dbt_project.yml

```diff
@@ -2,10 +2,14 @@ name: 'snowplow_web'
 version: '0.13.0'
 config-version: 2
 
-require-dbt-version: [">=1.3.0", "<2.0.0"]
+require-dbt-version: [">=1.4.0", "<2.0.0"]
 
 profile: 'default'
 
+dispatch:
+  - macro_namespace: dbt
```

Comment:
I see why you're saying we should remove naming overlap between `snowplow_utils` and `dbt` now, just to be on the safe side